parachain/runtime/snowbase/src/lib.rs

```diff
@@ -628,6 +628,7 @@ parameter_types! {
 	pub const MaxVoluntaryExitSize: u32 = 16;
 	pub const MaxAttestationSize: u32 = 128;
 	pub const MaxValidatorsPerCommittee: u32 = 2048;
+	pub const WeakSubjectivityPeriod: u32 = 27 * HOURS;
```

Comment:
Can't we go with 2 weeks? There are currently [489599](https://beaconscan.com/validators#active) validators, much more than the minimum of 262144 validators needed to sustain a 2-week period.

Also I suspect there are going to be many times when the light client won't be updated for a day or 2, due to operational reasons (like upgrades, etc). So don't want to trigger the WS check unnecessary.

Reply:
Also this line needs a comment with a brief explanation of what this period is.

Reply:
I am worried about a case where there is a major attack on the network and the vast majority of validators get slashed, and then the weak subjectivity threshold lowers to less than 2 weeks before we can update the threshold. It's unlikely, but not impossible.